### PR TITLE
Udash 0.8

### PR DESCRIPTION
**Pull request by ddworak**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1 +1 @@
-addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.11.0-M2")
\ No newline at end of file
+addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.11.0")
```
Comment (FreexD) [inline project/plugins.sbt:1]: when running `g8Test` multiple times and importing the project to intelliJ i get:
image

File: src/main/g8/default.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name = My Udash Application
 package = io.company.app
-scala_version = 2.12.6
-udash_version = 0.7.1
-sbt_version = 1.1.6
+scala_version = 2.12.8
```
Comment (FreexD) [inline src/main/g8/default.properties:3]: `sbt new UdashFramework/udash.g8` seems to still propose 2.12.6 as default scala version. Is that desired behaviour?

**Review by FreexD — CHANGES_REQUESTED** [review]

File: project/plugins.sbt (hunk 1 above)
Comment (ddworak) [reply project/plugins.sbt:1]: IntelliJ adds its files to target and makes g8 crazy. Should work fine for regular usage.

File: src/main/g8/default.properties (hunk 2 above)
Comment (ddworak) [reply src/main/g8/default.properties:3]: `sbt new UdashFramework/udash.g8` fetches master branch from this repo

**Review by FreexD — APPROVED** [review]
LGTM

**Pull request merged**